tools/RAiDER/cli/raider.py

```diff
@@ -410,12 +416,18 @@ def calcDelays(iargs=None):
         else:
             out_filename = w
 
+        # A dataset was returned by the above
+        # Dataset returned: Cube e.g. GUNW workflow
```

Comment:
Are these comments needed?

Reply:
`tropo_delay` has the [two different return types](https://github.com/dbekaert/RAiDER/blob/46da6a8ca231f6c99b9c43ee4632eccaf55067b9/tools/RAiDER/delay.py#L56-L57) and I was trying to figure out which of these two options the GUNW was using so I could add a breakpoint - would be nice to have a sign post. 

Can change language so long as going back it's clear that GUNW uses a particular branch.